tests/plugins/uploadwidget/uploadwidget.js

```diff
@@ -255,7 +255,7 @@
 			} );
 		},
 
-		'test replaceWith preserves selection placed before the widget': function() {
+		'test replaceWith preserves widget selection': function() {
```

Comment:
This test passes on `next` branch, so your fix isn't tested.

Reply:
It's more like this test only validates happy path. I see your point, I will write some regression test.